### ISSUE TITLE
updatepo.sh isn't working well...

### ISSUE THREAD

**Issue by ngosang** [minetest/minetest#2174]
I have found 2 problems when executing **updatepo.sh** for extract translation strings from source code:
1) The function **showOverlayMessage** translates the strings correctly but that strings are not detected by updatepo.sh. This should be easy to fix doing something like that:
```c++
void Game::showOverlayMessage(const wchar_t *msg, float dtime,
		int percent, bool draw_clouds)
{
	//wchar_t *text = wgettext(msg);
	draw_load_screen(msg, device, guienv, dtime, percent, draw_clouds);
	//delete[] text;
}
showOverlayMessage(wgettext("Resolving address..."), 0, 15);
```

2) **wstrgettext(...)** it's not detected by updatepo.sh. Example of usage:
```c++
	os		<< "button_exit[4," << (ypos++) << ";3,0.5;btn_sound;"
			<< wide_to_narrow(wstrgettext("Sound Volume")) << "]";
	os		<< "button_exit[4," << (ypos++) << ";3,0.5;btn_key_config;"
			<< wide_to_narrow(wstrgettext("Change Keys"))  << "]";
#endif
	os		<< "button_exit[4," << (ypos++) << ";3,0.5;btn_exit_menu;"
			<< wide_to_narrow(wstrgettext("Exit to Menu")) << "]";
	os		<< "button_exit[4," << (ypos++) << ";3,0.5;btn_exit_os;"
			<< wide_to_narrow(wstrgettext("Exit to OS"))   << "]"
```
In the source code I have found different ways to translate strings...
* gettext( ... )
* wgettext( ... )
* wide_to_narrow(wstrgettext( ... )) // this is not detected by updatepo.sh

... and I think it should be unified.

**Comment by ShadowNinja** [comment]
1. I'll add to #2176.
  2. Fixed by #2176.

**Comment by ShadowNinja** [comment]
efeb372e95e3bd056c0821c203fabc4e5d685c1f, 8bbf106379c67b1b32678264ac1703ab983f68ad
